src/Array.ts

```diff
@@ -1052,6 +1052,28 @@ export const zip = <A, B>(fa: Array<A>, fb: Array<B>): Array<[A, B]> => {
   return zipWith(fa, fb, tuple)
 }
 
+/**
+ * The function is reverse of `zip`. Takes an array of pairs and return two corresponding arrays
+ *
+ * @example
+ * import { unzip } from 'fp-ts/lib/Array'
+ *
+ * assert.deepEqual(unzip([[1, 'a'], [2, 'b'], [3, 'c']]), [[1, 2, 3], ['a', 'b', 'c']])
+ *
+ * @function
+ */
```

Comment:
@user753 thanks for this PR, could you please add the tag `@since 1.13.0` here?

Reply:
done